app/src/main/java/com/application/audiotesting/MainActivity.kt

```diff
@@ -86,20 +88,25 @@ class MainActivity : ComponentActivity() {
                                 )
                             }
                         }
-                        Column(
+
+                        val soundSources by audioPlayer.soundSources.collectAsState()
```

Comment:
this should be encapsulated behind a screen presenter at some point - not in this PR but a follow up.